hack/release/tkr-bom.yaml

```diff
@@ -1,428 +1,452 @@
       name: ubuntu
       version: "20.04"
       arch: amd64
   eu-west-2:
-  - id: ami-0a1ea980dc0e677e2
+  - id: ami-0d4df708ceea00ba1
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-02096b4f9a0558a2a
+  - id: ami-0cc22202b1fe14db6
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
   eu-west-3:
-  - id: ami-0f3d847bbc5a04d19
+  - id: ami-0416e28bd1712c97c
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-0fc1686196a449cbe
+  - id: ami-06c35743eea5234d9
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
   sa-east-1:
-  - id: ami-09bc44413b656c232
+  - id: ami-07c603e5bc4562c97
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-018113cf5656b3102
+  - id: ami-06323ef554739a647
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
   us-east-1:
-  - id: ami-0f0f0bb6426dfbe79
+  - id: ami-0ad65fb0aa8c5c7f7
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-0af8ffc12535aa02d
+  - id: ami-03714b1bf6617a560
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
   us-east-2:
-  - id: ami-08f4095e19c367152
+  - id: ami-05f4bd8f1bf7e0555
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-0fbe031f9d19aeac7
+  - id: ami-0afb50f648c9fe1e3
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
   us-gov-east-1:
-  - id: ami-0112cb5e809e4284d
+  - id: ami-05f5bdd4a771b9f9a
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-0383eb866523be1f5
+  - id: ami-0b9c6f322cb9b3167
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
   us-gov-west-1:
-  - id: ami-059061a8a5f3868cb
+  - id: ami-0b76187ec7dba7dbc
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-08c2d07dfdbc4beb3
+  - id: ami-0708fc229c8ede31b
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
   us-west-2:
-  - id: ami-0293f31158f5c72f3
+  - id: ami-017441d034a02c66f
     osinfo:
       name: amazon
       version: "2"
       arch: amd64
-  - id: ami-0c687784b0671fc49
+  - id: ami-0dd0327a3bfaa4dc8
     osinfo:
       name: ubuntu
       version: "20.04"
       arch: amd64
 azure:
-- sku: k8s-1dot21dot2-ubuntu-1804
+- sku: k8s-1dot22dot8-ubuntu-1804
   publisher: vmware-inc
   offer: tkg-capi
-  version: 2021.06.28
+  version: 2022.03.22
   thirdPartyImage: true
   osinfo:
     name: ubuntu
     version: "18.04"
     arch: amd64
-- sku: k8s-1dot21dot2-ubuntu-2004
+- sku: k8s-1dot22dot8-ubuntu-2004
   publisher: vmware-inc
   offer: tkg-capi
-  version: 2021.06.28
+  version: 2022.03.30
   thirdPartyImage: true
   osinfo:
     name: ubuntu
     version: "20.04"
     arch: amd64
 imageConfig:
-  imageRepository: projects.registry.vmware.com/tkg
+  imageRepository: projects-stg.registry.vmware.com/tkg
```

Comment:
This needs to be flipped back to `projects.registry.vmware.com/tkg`.

Reply:
Good catch!!!!